examples/text_labels_and_annotations/usetex_baseline_test.py

```diff
@@ -30,8 +30,10 @@ def __init__(self, *args, preview=False, **kwargs):
         super().__init__(*args, **kwargs)
 
     def draw(self, renderer):
-        with plt.rc_context({"text.latex.preview": self.preview}):
-            super().draw(renderer)
+        from matplotlib import _api  # internal, *do not use*
+        with _api.suppress_matplotlib_deprecation_warning():
```

Comment:
Since it is explicitly stated in this tutorial that this entire feature is deprecated and shouldn't be used, I wasn't too worried about also manually silencing this warning here, and just telling the user never to do the same.